setup.py

```diff
@@ -34,13 +36,14 @@
         "pyzmq~=22.3",
         "scikit-image>=0.16.2",
         "scipy>=1.4.1",
+        "zarr>=2.10",
     ],
     license="BSD",
     name="cellprofiler-core",
     package_data={"cellprofiler_core": ["py.typed"]},
     packages=setuptools.find_packages(exclude=["tests"]),
     python_requires=">=3.8",
     url="https://github.com/CellProfiler/core",
-    version="4.2.4",
+    version="5.0.0",
```

Comment:
`CellProfiler/setup.py` should be changed to `cellprofiler-core==5.0.0` in CellProfiler/CellProfiler#4660

Reply:
Done, thanks